uwblueprint/starter-code-v2
# Create release v1.9.1

## Notion ticket link

N/A



## Implementation description
* Followed [release instructions](https://github.com/uwblueprint/starter-code-v2#creating-a-release)
* Includes 2 commits from `main`: 6bd4b653a5b07bd7ea462af94d0d07cc2f328375 (README.md update), 44ce841fa986f65d0539babb5c3788f4c8234f95 (bug fixes)



## Steps to test
1. See test plan in #127, use stack combinations that allow testing of those items



## What should reviewers focus on?
* Anything missing from #127 and #128?


## Checklist
